app/scripts/controllers/transactions/index.js

```diff
@@ -324,9 +330,9 @@ export default class TransactionController extends EventEmitter {
     this.txStateManager.wipeTransactions(address);
   }
 
+  /* eslint-disable */
```

Comment:
why add `/* eslint-disable */` instead of adding the `@param` definition for `opts.securityAlertResponse`? 

I am thinking when `securityAlertResponse` isn't included for non-Blockaid builds, and with the jsdoc comment declared, this shouldn't affect the builds

Reply:
Code fencing is creating issue with it.

Reply:
can you just disable the linting rules conflicting here? And re-enable them right after the jsdoc?
something like this:
```js
  /* eslint-disable jsdoc/require-param, jsdoc/check-param-names */
  /**
   * Add a new unapproved transaction
   * @param {object} txParams - Standard parameters for an Ethereum transaction
   * @param {object} opts - Options
   * @param {string} opts.actionId - Unique ID to prevent duplicate requests
   * @param {string} opts.method - RPC method that requested the transaction
   * @param {string} opts.origin - Origin of the transaction request, such as the hostname of a dApp
   * @param {boolean} opts.requireApproval - Whether the transaction requires approval by the user
   * @param {object[]} opts.sendFlowHistory - Associated history to store with the transaction
   * @param {object} opts.swaps - Options specific to swap transactions
   * @param {boolean} opts.swaps.hasApproveTx - Whether this transaction required an approval transaction
   * @param {boolean} opts.swaps.meta - Additional metadata to store for the transaction
   * @param {TransactionType} opts.type - Type of transaction to add, such as 'cancel' or 'swap'
   * @returns {Promise<{transactionMeta: TransactionMeta, result: Promise<string>}>} An object containing the transaction metadata, and a promise that resolves to the transaction hash after being submitted to the network
   */
  /* eslint-enable jsdoc/require-param, jsdoc/check-param-names */
```

Reply:
@digiwand it fails due to code fencing... we run linting on fenced files when are not doing a dev build. Meaning that if we build with blockaid enabled (for example in flask) we needed the jsdoc param. But if you build for a build type without blockaid feature, then we get an linting error (jsdoc param does not exist). So the only way is to disable linting rules.

Reply:
oh didn't know we are running linting on the fenced files. makes sense now. ty for the info!